src/main/webapp/createPlaceGuide.html

```diff
@@ -40,7 +40,7 @@
             <input type = "checkbox" name = "deleteImage" value = "delete" id = "deleteImage">
             <br>
             <label for = "description">Description:</label><br>
-            <textarea name = "description" rows = "4" cols = "50" id = "description" pattern = "[A-Za-z0-9\s]+" title = "Enter alphanumeric string!"></textarea>
+            <textarea name = "description" rows = "4" cols = "50" id = "description" pattern = "[A-Za-z0-9,.\s]+" title = "You can only enter alphabets, numbers, periods, and comas!"></textarea>
```

Comment:
The alphabet is a set of letters, in this case, you meant letters, right?

Reply:
changed.